apps/financial-aid/web-veita/src/components/LogoMunicipality/index.tsx

```diff
@@ -21,34 +22,47 @@ interface LogoProps {
 const LogoMunicipality = ({ className }: LogoProps) => {
   const router = useRouter()
 
+  //According to design
+  const logoSize = 48
+
   //TODO when more muncipalities have been added to system
   const municipaliyId = router.query.sveitarfelag
     ? (router.query.sveitarfelag as string)
     : 'hfj'
 
-  const { data } = useQuery<MunicipalityData>(GetMunacipalityHomePageQuery, {
-    variables: {
-      input: {
-        id: municipaliyId,
+  const { data, loading, error } = useQuery<MunicipalityData>(
+    GetMunacipalityHomePageQuery,
+    {
+      variables: {
+        input: {
+          id: municipaliyId,
+        },
       },
+      fetchPolicy: 'no-cache',
+      errorPolicy: 'all',
     },
-    fetchPolicy: 'no-cache',
-    errorPolicy: 'all',
-  })
-
-  if (!data) {
-    return <LoadingDots />
-  }
+  )
 
   return (
-    <a
-      href={data.municipality.homePage}
-      target="_blank"
-      rel="noopener noreferrer"
-      className={cn({ [`${className}`]: true })}
+    <LoadingContainer
+      isLoading={loading}
+      loader={
+        <Box>
+          <SkeletonLoader display="block" height={logoSize} width={logoSize} />
+        </Box>
+      }
     >
-      <LogoSvg name={data.municipality.id} />
-    </a>
+      {data && (
```

Comment:
Do we need this check?

Reply:
We do, other wase the data.municipality complains